src/main/java/dev/cinema/models/ShoppingCart.java

```diff
@@ -0,0 +1,55 @@
+package dev.cinema.models;
+
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_cart")
+public class ShoppingCart {
+    @Id
+    private Long id;
+    @OneToMany (fetch = FetchType.EAGER)
```

Comment:
@YarikJ краще уникати EAGER. 
```suggestion
    @OneToMany
```
Спробуй пофіксити проблему з LazyInitializationException іншим шляхом. Наприклад через використання JOIN